linux_os/guide/services/ssh/ssh_server/sshd_set_max_sessions/rule.yml

```diff
@@ -0,0 +1,27 @@
+documentation_complete: true
+
+title: 'Set SSH MaxSessions limit'
+
+description: |-
+    The <tt>MaxSessions</tt> parameter specifies the maximum number of open sessions permitted
+    from a given connection. To set MaxSessions edit
+    <tt>/etc/ssh/sshd_config</tt> as follows: <pre>MaxSessions <b>sessions</b></pre>
+
+rationale: |-
+    To protect a system from denial of service due to a large number of concurrent
+    sessions, use the rate limiting function of MaxSessions to protect availability
+    of sshd logins and prevent overwhelming the daemon.
+
+severity: medium
+
+references:
+  cis@rhel8: 5.2.19
+  cis@sle15: 5.2.23
+
+ocil_clause: "MaxSessions is configured to the default of 10"
+
+ocil: |-
+    Run the following command to see what the max sessions number is:
+    <pre>$ sudo grep MaxSessions /etc/ssh/sshd_config</pre>
+    If properly configured, the output should be:
+    <pre>MaxSessions <sub idref="sshd_max_sessions" /></pre>
```

Comment:
Please make this `var_sshd_max_sessions`

Reply:
Resolved in 0c6d42e